packages/dbos-cloud/cloudutils.ts

```diff
@@ -77,6 +77,10 @@ export function credentialsExist(): boolean {
   return fs.existsSync(`./${dbosEnvPath}/credentials`);
 }
 
+export function deleteCredentials() {
+  fs.unlinkSync(`./${dbosEnvPath}/credentials`);
```

Comment:
what happens if the file doesn't exist? Throw an exception?

Reply:
We check if the file exists before calling this function